cfme/storage/manager.py

```diff
@@ -109,10 +109,14 @@ class StorageManagerDetailsView(StorageManagerView):
     @property
     def is_displayed(self):
         expected_title = '{} (Summary)'.format(self.context['object'].name)
-
+        active_location = (
```

Comment:
1. it would be great to get rid of how many times this is implemented, optional here.

2.  I think it would be cleaner to just use expected_title, and format into it. Then we don't have multiple references to the context object's name attribute, or extra names.
```
expected_title = '{}{}'.format(self.context['object'].name, 
                                             ' (Summary)' if self.browser.product_version <= '5.10' else '')
```

Reply:
@mshriver as per new implementation in `breadcrumb` we are not getting `(Summary)` and title is same; so we need two separate references.
```
5.10
title = 'obj_name (summary)'
breadcrumb= 'obj_name (summary)'

5.11
title = 'obj_name (summary)'
breadcrumb= 'obj_name'
```